lua/pluginList.lua

```diff
@@ -324,4 +324,38 @@ return packer.startup(function()
          require("mappings").fugitive()
       end,
    }
+
+   use {
```

Comment:
Hmmmm, I do think we may be missing some of the functionality of LSPSaga?

Like autodoc hover, I'm not sure if anything we have does this already?